src/domain/event/__tests__/EditEventPage.test.tsx

```diff
@@ -124,3 +134,164 @@ test('returns to create occurrences page when it should after saving', async ()
     );
   });
 });
+
+describe('Event price section', () => {
+  test('price field is accessible only when isFree field is not checked', async () => {
+    render(<EditEventPage />, { mocks: editMocks });
+    await screen.findByLabelText(/Tapahtuma on ilmainen/);
+
+    expect(screen.getByLabelText(/Tapahtuma on ilmainen/)).toBeChecked();
+    expect(screen.getByLabelText(/Hinta/)).toHaveAttribute('disabled');
+    expect(screen.getByLabelText(/Lisätiedot/)).toHaveAttribute('disabled');
+
+    userEvent.click(screen.getByLabelText(/Tapahtuma on ilmainen/));
+
+    expect(screen.getByLabelText(/Tapahtuma on ilmainen/)).not.toBeChecked();
+    expect(screen.getByLabelText(/Hinta/)).not.toHaveAttribute('disabled');
+    expect(screen.getByLabelText(/Lisätiedot/)).not.toHaveAttribute('disabled');
```

Comment:
`expect().toBeDisabled` could be used here

Reply:
yep!